tests/end2end/features/test_editor_bdd.py

```diff
@@ -64,3 +64,21 @@ def set_up_editor(quteproc, server, tmpdir, text):
 def set_up_editor_empty(quteproc, server, tmpdir):
     """Set up editor.command to a small python script inserting empty text."""
     set_up_editor(quteproc, server, tmpdir, "")
+
+
+@bdd.when(bdd.parsers.parse(
+    'I set up a fake editor returning "{text}" after {t}s'))
+def set_up_editor_delay(quteproc, server, tmpdir, text, t):
+    """Set up editor.command to a small python script inserting a text."""
+    script = tmpdir / 'script.py'
+    script.write(textwrap.dedent("""
+        import sys
+        import time
+
+        time.sleep({t})
```

Comment:
This sounds like a recipe for flaky tests - can't we save the editor somewhere, and have another `@bdd.when` function quitting the editor somehow (e.g. by making the Python script exit cleanly on `SIGUSR1` or so)?